src/random.cr

```diff
@@ -90,6 +90,20 @@ module Random
     end
   end
 
+  # Returns a random `Float64` which is greater than or equal to 0 and less than *max*.
+  #
+  # ```
+  # Random.new.rand(3.5)   # => 2.88938
+  # Random.new.rand(10.725) # => 7.70147
+  # ```
+  def rand(max : Float) : Float64
+    if max > 0
+      next_u32 * (1 / (UInt32::MAX.to_f64 + 1)) * max
```

Comment:
This:

```crystal
next_u32 * (1 / (UInt32::MAX.to_f64 + 1))
```

is almost the same as calling `next_float`. Can we maybe use `next_float`? Why we need to add 1 to UInt32::MAX here? (or why not in the other case)